expo/tests/contexts/act_techinfo_context.py

```diff
@@ -0,0 +1,46 @@
+from tests.factories.gbe_factories import (
```

Comment:
Stopping here... commute almost over... hopeful to pickup here.  I want to read this carefully to get a clue on contexts.